data/src/test/java/org/apache/iceberg/TestSplitScan.java

```diff
@@ -60,7 +79,7 @@ public TestSplitScan(String format) {
   }
 
   @Before
-  public void setup() throws IOException {
+  public void before() throws IOException {
```

Comment:
What rule required this change? I usually like to have descriptive names for `@Before` and `@After` methods. Is the name required to be `before`? (Granted, the original name, `setup` wasn't very descriptive either.)

Reply:
It seems that I can name it anything other than setup/teardown. Below is the error

> Task :iceberg-data:checkstyleTest
[ant:checkstyle] [ERROR] /Users/rratti/code/iceberg/data/src/test/java/org/apache/iceberg/TestSplitScan.java:82: Test setup/teardown methods are called before(), beforeClass(), after(), afterClass(), but not setUp, teardown, etc. [RegexpSinglelineJava]